atlascope/core/rest/endpoints/dataset_endpoints.py

```diff
@@ -57,3 +69,17 @@ def subimage(self, request, pk):
         subimage = original.subimage(**serializer.validated_data)
 
         return Response(DatasetSerializer(subimage).data, status=status.HTTP_201_CREATED)
+
+    @action(detail=True, methods=['GET'], renderer_classes=(PassthroughRenderer,))
+    def download(self, request, pk):
+        dataset = self.get_object()
+        try:
+            path = dataset.content.path
+        except Exception:
```

Comment:
Is there a more precise `Exception` class we can catch here?

Reply:
Yes, updated in e555282